extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -131,9 +136,19 @@ private JibContainer containerize(ApplicationInfoBuildItem applicationInfo, Cont
     }
 
     private Containerizer createContainerizer(ContainerImageConfig containerImageConfig,
+            ContainerImageInfoBuildItem containerImage,
             ApplicationInfoBuildItem applicationInfo, boolean pushRequested) {
         Containerizer containerizer;
-        ImageReference imageReference = getImageReference(containerImageConfig, applicationInfo);
+        ImageReference imageReference = getImageReference(containerImageConfig, containerImage, applicationInfo);
+
+        if (!containerImage.getAdditionalTags().isEmpty()) {
+            containerImage.getAdditionalTags().forEach(additionalTag -> {
```

Comment:
Can we get rid of the these lambdas please? We really try to avoid them in cases where they bring no real benefit (as they do have a small runtime cost), like here. A simple for loop would be just fine.